web/src/app/shared/components/pagination/pagination.component.ts

```diff
@@ -75,4 +80,9 @@ export class PaginationComponent implements OnInit, OnChanges {
     this.pageSelected = page;
     this.changePage.emit(page);
   }
+
+  filter(v) {
```

Comment:
please add a type 🙂 and I think it would be nice to have a more meaningful name than 'v'.